app/App/Dump.hs

```diff
@@ -144,6 +144,10 @@ dumpSegment hOut i filename segment = do
               forM_ (word64ToList idx w64 []) $ \w32 -> do
                 let ipString = w32 & word32ToIpv4 & ipv4ToString
                 liftIO $ IO.hPutStrLn hOut $ ipString <> replicate (16 - length ipString) ' ' <> "(" <> show w32 <> ")"
+          Just (Known F.Bool) ->
+            forM_ (LBS.chunkBy 1 (segment ^. the @"payload")) $ \bs -> do
+              let w :: Bool = G.runGet G.get (LBS.take 1 (bs <> LBS.replicate 1 0))
```

Comment:
Why do we need to `LBS.take 1 (bs <> LBS.replicate 1 0)` and can't just `G.get bs`?

Reply:
I haven't the faintest. That was the pattern for both `Int8` and `Word8`, so I (perhaps incorrectly) copied that pattern.

Reply:
I looked a bit around. Looks like it is a way to compensate with the case where `bs` is exhausted, but you still want to read from it. Then it will read this "default" value that we've appended (in this case it is `0`).
Still, don't know why we need to `take`...